main.py

```diff
@@ -0,0 +1,47 @@
+import random;
```

Comment:
Don't forget to update README file to give other developers a good brief of what this project does